### PR TITLE
[AIFD] Change in title

### PR DESCRIPTION
**Pull request by D047539**

### CODE REVIEW

**Comment by albertmink** [comment]
Requires only review from @SAP/abap-file-formats-ux

**Review by albertmink — APPROVED** [review]
LGTM

File: file-formats/aifd/type/zif_aff_aifd_v1.intf.abap [hunk 1]
```diff
@@ -23,8 +23,8 @@ INTERFACE zif_aff_aifd_v1
       "! Header
       "! $required
       header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
-      "! <p class="shorttext">AIF Interfaces</p>
-      "! AIF interfaces
+      "! <p class="shorttext">Interfaces</p>
+      "! Interfaces
       "! $required
       aif_interfaces TYPE ty_aif_interfaces,
```
Comment (wurzka) [inline file-formats/aifd/type/zif_aff_aifd_v1.intf.abap:29]: The name of the component still contains `aif`. We usually recommend having the name and the title in sync. For me, it is fine here. However, I want to make sure, that in the JSON field (for example in https://github.com/SAP/abap-file-formats/blob/main/file-formats/aifd/examples/z_aff_example_aifd.aifd.json), the component is still named `aifInterface`.
If the presence of `aif` is ok for you, we can merge this change

**Review by wurzka — COMMENTED** [review]
Just one question, see my comment

File: file-formats/aifd/type/zif_aff_aifd_v1.intf.abap (hunk 1 above)
Comment (D047539) [reply file-formats/aifd/type/zif_aff_aifd_v1.intf.abap:29]: Unfortunately, AIF is not an official abbreviation for our product, so it is not allowed to be seen on the UI. However, as a component name in the background it should be ok from our point of view and gives a little more detail about what is displayed and maintained.

Comment (wurzka) [reply file-formats/aifd/type/zif_aff_aifd_v1.intf.abap:29]: But that's my point. It is not only in the background. It is in the JSON file, part of the "official" AFF. So, if `aif` should not be seen, I suggest removing it at least from the component name.

Comment (D047539) [reply file-formats/aifd/type/zif_aff_aifd_v1.intf.abap:29]: I need to align with our UA developer and PO how to proceed with the abbreviation AIF

Comment (D047539) [reply file-formats/aifd/type/zif_aff_aifd_v1.intf.abap]: We will stay with abbreviation AIF

**Comment by albertmink** [comment]
No changes -> closing the issue

**Pull request closed without merging**